src/commonMain/kotlin/fr/acinq/lightning/crypto/LocalKeyManager.kt

```diff
@@ -10,7 +10,7 @@ import fr.acinq.lightning.transactions.Transactions
 
 data class LocalKeyManager(val seed: ByteVector, val chainHash: ByteVector32) : KeyManager {
 
-    private val master = DeterministicWallet.generate(seed)
+    val master = DeterministicWallet.generate(seed)
```

Comment:
We could instead make ` privateKey(keyPath: KeyPath)` and `fun publicKey(keyPath: KeyPath)` public ?

Reply:
Fixed in da16e221

It is more consistent to make the `privateKey` methods public.